infrastructure/admin.py

```diff
@@ -271,17 +271,45 @@ class Meta:
 
 @admin.register(Fuel)
 class FuelAdmin(admin.ModelAdmin):
-    pass
+    save_on_top = True
+    list_display = (
+        'name',
+        'fuel_category'
+    )
+    list_filter = (
+        'name',
+        'fuel_category'
+    )
+    search_fields = ('name', 'fuel_category__name')
 
 
 @admin.register(FuelCategory)
 class FuelCategoryAdmin(admin.ModelAdmin):
-    pass
+    save_on_top = True
+    list_filter = (
+        'name',
+    )
+    search_fields = ('name',)
 
 
 @admin.register(OwnerStake)
 class OwnerStakeAdmin(admin.ModelAdmin):
-    pass
+    save_on_top = True
+    form = ProjectOwnerStakeForm
+    list_display = (
```

Comment:
Can you make the first item here `'id'`, so that clicking on the `Organization`'s name doesn't take the user to the `OwnerStake` page?